site/docs/v1/tech/themes/index.adoc

```diff
@@ -661,6 +671,29 @@ The email address that was passed as a URL parameter. This is the email address
 [field]#emailSent# [type]#[Boolean]#::
 A boolean that indicates if the verification email was re-sent or not.
 
+=== Email verification required
```

Comment:
`Email verification re-send` should be `Email verification re-sent` (t on the end) to match what is in the UI

Reply:
Couldn't put it higher, sorry.

Reply:
Thanks Dan, I fixed those typos. I'll merge in a bit.